bin/brew

```diff
@@ -19,10 +19,12 @@ symlink_target_directory() {
   quiet_cd "$directory" && quiet_cd "$target_dirname" && pwd -P
 }
 
+# Unset functions that override Bash builtins
+# Enable all Bash builtins
```

Comment:
Sorry: to be more explicit: can these be combined into a single statement? Ideally about "why" this is being done rather than "what" (as the code makes it relatively apparent).

Reply:
No worries. I gave it the best shot. Feel free to make suggestions if the new version is not to your liking.